### PR TITLE
TASK: Refactor TYPO3.Media to use PSR-4

### PR DESCRIPTION
**Pull request by johannessteu**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
         "typo3/flow": "*",
         "typo3/imagine": "*",
         "typo3/kickstart": "*",
+        "typo3/form": "*",
```
Comment (johannessteu) [inline composer.json]: Due to our ComposerManifest Script. NodeTypes has a dependency on form

Comment (aertmann) [inline composer.json]: ah okay, well this needs to be fixed in 2.3 as a bugfix and upmerged then

Comment (johannessteu) [inline composer.json]: @aertmann https://github.com/neos/neos-development-collection/pull/1196

**Comment by johannessteu** [comment]
This one is important to get the tests green 
https://github.com/neos/BuildEssentials/pull/21

File: composer.json [hunk 2]
```diff
@@ -33,12 +34,12 @@
         "psr-4": {
             "Neos\\Diff\\": [
                 "Neos.Diff/Classes"
+            ],
+            "TYPO3\\Media": [
```
Comment (Benjamin-K) [inline composer.json]: PSR-4 namespace should end with \\

**Comment by robertlemke** [comment]
Closing this pull request because the PSR-4 migration will be done by @kitsunet as part of the vendor namespace change.

**Pull request closed without merging**